getting_started/dev-environment/ubuntu.md

```diff
@@ -0,0 +1,42 @@
+# Instructions for Ubuntu
+
+Run the following to install most of the needed packages:
+
+```shell
+$ sudo apt-get install python python-setuptools python-dev build-essential \
+                       python-software-properties python-pip vagrant virtualbox
+```
```

Comment:
This should be written as one line:
```markdown
    ```shell
    $ sudo apt-get install python python-setuptools python-dev build-essential python-software-properties python-pip vagrant virtualbox
    ```
```